templates/weno/admin.html.twig

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Remove this script since not seeing it used anywhere.

Reply:
removed it for future work of moving the weno admin over to twig. Really getting into this Symfony stuff.